src/restapi/fConfig.ts

```diff
@@ -35,6 +35,7 @@ export async function fSubmitOrUpdateZendeskConfigSubmit(req: Request, res: Resp
         const targetID = cValues.zd_target_id;
         const zdOauth2AccessToken = cValues.zd_oauth_access_token;
         const storeValues = call.values as AppConfigStore;
+        storeValues.zd_url = storeValues.zd_url.replace(/\/$/, '');
```

Comment:
What validation do we have that verifies that this is a valid zendesk URL? Can we ping zendesk's API to verify that the URL is valid?

Reply:
Note that if there is more than one trailing slash, it will only remove the last one.

Reply:
I updated the replace statement to remove all ending `/`'s

Reply:
I haven't added any validation for the provided URL.  What if I just ping the login `https://mattermost1616716488.zendesk.com/access/unauthenticated` when responds with status `200` if the site exists?

Reply:
Yeah that sounds good

Reply:
![image](https://user-images.githubusercontent.com/7575921/128244934-1d495798-6df8-43ef-927e-c3e685104e37.png)

@mickmister, I added a check for a valid URL.  This is pretty nice because the modal hyperlinks the URL in the error :)